db/memtable_list.cc

```diff
@@ -319,12 +319,95 @@ void MemTableList::RollbackMemtableFlush(const autovector<MemTable*>& mems,
   imm_flush_needed.store(true, std::memory_order_release);
 }
 
+namespace {
+uint64_t FindMinPrepLogReferencedByMemTable(
```

Comment:
I am not sure if this is a good idea to move this function memtable_list. This has resulted into duplicating the logic into TEST_FindMinPrepLogReferencedByMemTable when needed by tests. In long term this could result into having tests testing a code that will not be run in production.